altapay.php

```diff
@@ -371,23 +371,24 @@ function altapay_meta_box_side( $post_or_order_object ) {
 				if ( $status === 'released' ) {
 					echo '<strong>' . __( 'Payment Released.', 'altapay' ) . '</strong>';
 				} else {
-					$charge = $reserved - $captured - $refunded;
-					if ( $charge <= 0 ) {
-						$charge = 0.00;
-					}
+					$charge        = max( 0, $reserved - $captured );
```

Comment:
What is there has been refunds?

Reply:
@emicha I've updated the chargeable amount calculations to include refunds as well.

Example:
Reserved = 1500
Captured = 500
Refunded = 500

Chargeable = Reserved - Captured + Refunded
Chargeable = 1500 - 500 + 500 = 1500